Update Yardian docs for new sensors (#40940)
home-assistant/home-assistant.github.io: source/_integrations/yardian.markdown

ha_domain: yardian
ha_platforms:
  - binary_sensor
  - switch
ha_integration_type: integration
---

The **Yardian** {% term integration %} allows you to control your [Yardian Smart Sprinkler Controller](https://yardian.com/products/yardian-pro-smart-sprinkler-controller/).

There is currently support for the following platforms within Home Assistant:

- Switch - Allows you to view the status of zones and control them.
- Binary sensor - Shows watering status along with standby and freeze prevent diagnostics. Per-zone enabled diagnostics are provided but disabled by default.

{% include integrations/config_flow.md %}

During the configuration, you will have to manually set the **Host** and the **Access Token**. You can find them inside your [Yardian App](https://yardian.com/app/).

![Yardian Host/Token Location](/images/integrations/yardian/yardian_config_flow.jpg)

## Supported functionality

The **Yardian** integration provides the following entities.

### Binary sensors

- **Watering running**: Is `on`  when a zone is currently irrigating.
- **Standby**:  Is `on` when the controller is in standby mode.
- **Freeze prevent**: Turns on when the controller enables freeze prevention.

> - **Zone enabled**: `On` if a zone is enabled. These entities are disabled by default and created per zone

Reviewer: ```suggestion
- **Zone enabled**: `On` if a zone is enabled. These entities are disabled by default and created per zone.
```

  - '@h3l1o5'
ha_domain: yardian
ha_platforms:
  - binary_sensor
  - sensor
  - switch
ha_integration_type: integration
---

The **Yardian** {% term integration %} allows you to control your [Yardian Smart Sprinkler Controller](https://yardian.com/products/yardian-pro-smart-sprinkler-controller/).

There is currently support for the following platforms within Home Assistant:

- Switch - Allows you to view the status of zones and control them.
- Binary sensor - Shows watering status along with standby and freeze prevent diagnostics. Per-zone enabled diagnostics are provided but disabled by default.
- Sensor - Reports rain delay, the number of active zones, and diagnostic values such as sensor delay, water hammer duration, and the controller region. Most diagnostic sensors are disabled by default.

{% include integrations/config_flow.md %}

During the configuration, you will have to manually set the **Host** and the **Access Token**. You can find them inside your [Yardian App](https://yardian.com/app/).

![Yardian Host/Token Location](/images/integrations/yardian/yardian_config_flow.jpg)

> ## Entities

Reviewer: ```suggestion
## Supported functionality

The **Yardian** integration provides the following entities.
```

### Binary sensors

- **Watering running** – Indicates when any zone is currently irrigating.
- **Standby** – Diagnostic binary sensor showing whether the controller is in standby mode.
- **Freeze prevent** – Diagnostic binary sensor that turns on when the controller enables freeze prevention.

> - **Zone enabled** – Diagnostic binary sensors created per zone that mirror the zone enablement flags. These entities are disabled by default.

Reviewer: ```suggestion
- **Watering running**: Is `on`  when a zone is currently irrigating.
- **Standby**:  Is `on` when the controller is in standby mode.
- **Freeze prevent**: Turns on when the controller enables freeze prevention.
- **Zone enabled**: On if a zone is enabled. These entities are disabled by default and created per zone
```

### Sensors

- **Rain delay** – Remaining rain delay reported in seconds.
- **Active zone count** – The number of zones currently running.
- **Sensor delay** – Diagnostic duration in seconds (disabled by default).
- **Water hammer duration** – Diagnostic duration in seconds (disabled by default).

> - **Region** – Diagnostic text describing the configured region (disabled by default).

Reviewer: ```suggestion
- **Rain delay**: Remaining rain delay reported in seconds.
- **Active zone count**: The number of zones currently running.
- **Sensor delay**: Duration in seconds (disabled by default).
- **Water hammer duration**: Duration in seconds (disabled by default).
- **Region**: Text describing the configured region (disabled by default).
```

  - '@h3l1o5'
ha_domain: yardian
ha_platforms:
  - binary_sensor

>   - sensor

Reviewer: ```suggestion
```

  - switch
ha_integration_type: integration
---

The **Yardian** {% term integration %} allows you to control your [Yardian Smart Sprinkler Controller](https://yardian.com/products/yardian-pro-smart-sprinkler-controller/).

There is currently support for the following platforms within Home Assistant:

- Switch - Allows you to view the status of zones and control them.
- Binary sensor - Shows watering status along with standby and freeze prevent diagnostics. Per-zone enabled diagnostics are provided but disabled by default.

> - Sensor - Reports rain delay, the number of active zones, and diagnostic values such as sensor delay, water hammer duration, and the controller region. Most diagnostic sensors are disabled by default.

Reviewer: ```suggestion
```

{% include integrations/config_flow.md %}

During the configuration, you will have to manually set the **Host** and the **Access Token**. You can find them inside your [Yardian App](https://yardian.com/app/).

![Yardian Host/Token Location](/images/integrations/yardian/yardian_config_flow.jpg)

## Supported functionality

The **Yardian** integration provides the following entities.

### Binary sensors

- **Watering running**: Is `on`  when a zone is currently irrigating.
- **Standby**:  Is `on` when the controller is in standby mode.
- **Freeze prevent**: Turns on when the controller enables freeze prevention.

> - **Zone enabled**: On if a zone is enabled. These entities are disabled by default and created per zone

Reviewer: ```suggestion
- **Zone enabled**: `On` if a zone is enabled. These entities are disabled by default and created per zone
```

### Sensors

> - **Rain delay**: Remaining rain delay reported in seconds.

Reviewer: what is a rain delay?

Author: https://www.yardian.com/blogs/guides/skip-rules/#:~:text=the%20rain%20sensor-,Rain%20Sensor%20Skip,your%20choice%20will%20be%20skipped.

- **Active zone count**: The number of zones currently running.

> - **Sensor delay**: Duration in seconds (disabled by default).

Reviewer: What do you mean by sensor delay? Delay before/between what?

Author: Should be zone delay - have fixed.
https://www.yardian.com/blogs/guides/zone-delay/

> - **Water hammer duration**: Duration in seconds (disabled by default).

Reviewer: what do you mean by water hammer duration? How long it takes for the pressure surge to stop after the wtaer has been turned of?

Author: https://www.yardian.com/blogs/guides/water-hammer-reduction/

> - **Region**: Text describing the configured region (disabled by default).

Reviewer: ```suggestion
```
